test/integration/next-image-new/app-dir/test/static.test.ts

```diff
@@ -238,7 +238,7 @@ describe('Build Error Tests', () => {
           "Module not found: Can't resolve '../../public/foo/test-rect-broken.jpg"
         )
         // should contain the importing module
-        expect(stderr).toContain('./app/static-img/page.js')
+        expect(stderr).toContain('app/static-img/page.js')
```

Comment:
What's the logged path in Turbopack? I'd rather have the previous assertion forked on Turbopack because this may mask that we print an unreadable absolute path. Same for the other instances where we stop asserting a relative path.

Reply:
probably `[project]/app/static-img/page.js`

Reply:
Can we encode this in tests please to make sure this is actually the case